generators/server/needle-api/needle-server-gateway.js

```diff
@@ -70,7 +70,7 @@ module.exports = class extends needleBase {
                                     |    "Type": "LeastConnection"
                                     |  },
                                     |  "ReRoutesCaseSensitive": false,
-                                    |  "UpstreamPathTemplate": "/services/${microserviceName}/api/${entityName}/{everything}",
+                                    |  "UpstreamPathTemplate": "/${microserviceName}/api/${entityName}/{everything}",
```

Comment:
Are you sure removes "services" works ? I have verify in angular service the url start by services 
https://github.com/jhipster/generator-jhipster/blob/main/generators/entity-client/templates/angular/src/main/webapp/app/entities/entity.service.ts.ejs#L48

Reply:
@nicolas63  sure ,works.

Reply:
with '/services' , when Edit,Delete,or View  Item   rise this Error:
"status":404,"statusText":"Not Found","url":"http://localhost:5000/services/microservice/api/bars/1","ok":false,"name":"HttpErrorResponse","message":"Http failure response for http://localhost:5000/services/microservice/api/bars/1: 404 Not Found",

Reply:
Can you share your `entity.service` ?